sky/packages/sky/lib/src/widgets/media_query.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'basic.dart';
+import 'framework.dart';
+import 'transitions.dart';
+
+enum Orientation { portrait, landscape }
+
+class MediaQueryData {
+
+  const MediaQueryData({ this.size });
+
+  final Size size;
+
+  Orientation get orientation {
+    return screenSize.width > screenSize.height ? Orientation.landscape : Orientation.portrait;
+  }
+
+  String toString() => '$size $orientation';
```

Comment:
I'd probably do something like '$runtimeType($size, $orientation)', so you can more easily tell what it is if you dump it without knowing what you're dumping (e.g. in a big Map or List).